### ISSUE TITLE
GitOps does not automatically log out after token session has expired

### ISSUE THREAD

**Issue by rokshana-b** [weaveworks/weave-gitops#1918]
**Describe the bug**

After logging in and staying inactive for a while, GitOps does not automatically log out after token session has expired. It shows error messages saying either `Authentication Required` or `Unexpected Token`. Page requires either refresh or navigation to a different page to trigger redirection to the Login Page.

**Screenshots**

﻿![Screen Shot 2022-04-13 at 16.15.38.png](https://images.zenhubusercontent.com/606389351ead0c83798a9a73/f135f23c-4cf1-49d1-8f5d-6100614f681d)

<img width="1792" alt="image" src="https://user-images.githubusercontent.com/76788247/163278666-c9a0dd27-e450-4c5f-a5ad-04f20b0ab949.png">

**Expected behavior**

When a user token session expires, the UI should automatically log out instead of displaying error messages. Log out should not be triggered only after a page refresh. 

**Is this a UI bug or a server bug?**
- [x] UI
- [ ] Server

**What is the severity of the bug**

- [ ] `severity/Critical`: Weave GitOps is crashing or experiencing data loss, the UI is inaccessible or a key feature is unusable. There is no known workaround
- [ ] `severity/Major`: Weave Gitops functionality is broken, there is a workaround, but the workaround requires significant effort
- [x] `severity/Minor`: Weave Gitops functionality is broken, but there is a fairly straightforward workaround
- [ ] `severity/Low`: Doesn’t affect primary flow/functionality but would be good to fix

**Environment**
 - gitops: v0.7.0
 - How you deployed the Weave GitOps server: Helm Chart
 - kubernetes:  [e.g. 1.20.4]
    - [x] KinD - _version_]
    - [x] other - standing GKE cluster env setup for testing

**To Reproduce**

1. Login to UI dashboard
2. Stay inactive on the page for a while
3. See error message displayed on the page

**Comment by rokshana-b** [comment]
Duplicate of #1894
